packages/curvenote-cli/src/session/session.ts

```diff
@@ -109,9 +109,8 @@ export class Session implements ISession {
         session.setUserToken({ token, decoded });
         await session.refreshSessionToken();
       }
-
-      await session.configure();
     }
+    await session.configure();
```

Comment:
Yeah, I think one issue here is now when unauthenticated, it's going to fall back to the default config, Which will always be prod probably OK for now, but it could cause issues for non-curvenote public deployment.